src/validate.ts

```diff
@@ -189,6 +194,9 @@ export function response(compiledPath: CompiledPath | undefined,
   if (response === undefined) {
     response = operation.responses['default'];
   }
-
+  // make Date or ObjectId to string
```

Comment:
Thanks! For the reason of timezone, I have to finish this pr after several hours.